src/clr-angular/data/datagrid/_datagrid.clarity.scss

```diff
@@ -215,24 +237,54 @@
                 line-height: 1rem;
             }
         }
+
+        .datagrid-action-overflow {
+          z-index: map-get($clr-layers, datagrid-popover);
+        }
     }
 
-    .datagrid-row-flex {
+    .datagrid-row-sticky {
+        background-color: $clr-table-bgcolor;
+        display: flex;
+        flex-wrap: nowrap;
+        position: sticky;
+        left: 0;
+    }
+
+    .datagrid-row-scrollable {
         flex: 1 1 auto;
         display: flex;
-        flex-flow: row nowrap;
+        flex-flow: column nowrap;
 
-        .datagrid-row-detail {
-            &.datagrid-container {
-                padding-top: 0;
+        &.is-replaced {
+            flex-direction: row;
+
+            .datagrid-scrolling-cells {
+                flex: 0 0 auto;
             }
+        }
+    }
 
+    .datagrid-row-flex {
+        flex: 1 1 auto;
+        display: flex;
+        flex-flow: column nowrap;
+
+        .datagrid-row-detail {
+            display: fles;
```

Comment:
`display: fles`? 😛 

But seriously, how is it even rendering correctly right now?

Reply:
Will fix and confirm the functionality.